cms/static/js/spec/views/active_video_upload_list_spec.js

```diff
@@ -174,42 +177,71 @@ define(
                             // 2.0, the latest version of jasmine-ajax (mock-ajax.js) does have the
                             // necessary support.
 
-                            _.each(
-                                [
-                                    {
-                                        desc: 'completion',
-                                        responseStatus: 204,
-                                        statusText: ActiveVideoUpload.STATUS_COMPLETED,
-                                        progressValue: 1,
-                                        presentClass: 'success',
-                                        absentClass: 'error'
-                                    },
-                                    {
-                                        desc: 'failure',
-                                        responseStatus: 500,
-                                        statusText: ActiveVideoUpload.STATUS_FAILED,
-                                        progressValue: 0,
-                                        presentClass: 'error',
-                                        absentClass: 'success'
-                                    }
-                                ],
+                            _.each([true, false],
+                                function(viewRefresh) {
+                                    var refreshDescription = viewRefresh ? ' (refreshed)' : ' (not refreshed)';
+                                    _.each(
```

Comment:
@mushtaqak I incorrectly thought in your previous comment for line 195 you approved of this.

I am using `_each` here to avoid repetition while defining `subCases`. All other attributes of the sub-case are same except `viewRefresh` and `desc`.

I am still okay to change it if it is needed. Please let me know.